platforms/tktrex/backend/bin/parser.ts

```diff
@@ -97,8 +97,18 @@ const run = async (): Promise<void> => {
         ...parserConfig,
         errorReporter: (e: any) => {
           const entryNature = e.html.nature.type ?? 'failed';
+          const fixtureFolderPath = path.resolve(
+            process.cwd(),
+            'parsers/__tests__/fixtures'
+          );
+
+          // ensure fixtures folder path exists
+          if (!fs.existsSync(fixtureFolderPath)) {
+            fs.mkdirSync(fixtureFolderPath, { recursive: true });
+          }
+
```

Comment:
As with youtube, I'd like the code to create the subfolders too
Something like

```js
const subfolders = ["htmls/native","htmls/profile","htmls/search"]
for (const sf of subfolders){
    const fullPath = path.resolve(fixtureFolderPath, sf));
    if (!fs.existsSync(fullPath))
                 fs.mkdirSync(fullPath, {recursive: true});
}
```

**In addition** to that, tktrex also need the download folder (`downloads/{thumbnail,video}`)

Reply:
Also here I added the `entryNature` to `fixtureFolderPath`

> In addition to that, tktrex also need the download folder (downloads/{thumbnail,video})

While this one will be addressed in another PR